pages/recording/[id].tsx

```diff
@@ -34,11 +98,41 @@ function Recording({ getAccessibleRecording }: PropsFromRedux) {
   if (!uploadComplete && recording.isInitialized === false && !isTest() && recording.userId) {
     return <Upload onUpload={() => setUploadComplete(true)} />;
   } else {
-    return <DevTools uploadComplete={uploadComplete} />;
+    return (
+      <>
+        {head}
+        <DevTools uploadComplete={uploadComplete} />
+      </>
+    );
   }
 }
 
 const connector = connect(null, { getAccessibleRecording });
 type PropsFromRedux = ConnectedProps<typeof connector>;
 
-export default connector(Recording);
+const ConnectedRecordingPage = connector(RecordingPage);
+
+export async function getStaticProps({ params }: { params: { id: string } }) {
```

Comment:
Might be better to use the correct type https://nextjs.org/docs/basic-features/data-fetching#getstaticprops-static-generation for context and not raw type.